packages/gatsby-plugin-nginx/src/nginx-generator.ts

```diff
@@ -217,7 +217,7 @@ export function convertToRegExp(path: string) {
     // allows '<path>/*' to serve '<path>' (without trailing slash)
     // this is necessary because we are now removing trailing slashes from incoming requests
     // so exact matches can work when there is a trailing slash
-    converted = converted.slice(0, -catchAll.length) + '(?:/(.*))?'
+    converted = `${converted.slice(0, -catchAll.length)}(?:/(.*))?`
```

Comment:
This is not due to this PR, but what's up with [this ESLint comment](https://github.com/vtex/faststore/runs/5645500405?check_suite_focus=true#step:5:10)? I thought this change would have fixed it.
😕